[ticket-2] adds baseline index file and preliminary css

justinw248 opened the pull request:
Adds landing page and early css.

justinw248 committed:
adds baseline index file and preliminary css

zefrof commented on css/app.css:
```diff
@@ -0,0 +1,93 @@
+    margin: 0px;
+}
+
+html {
+
+}
```
```suggestion
```

zefrof commented on css/app.css:
```diff
@@ -0,0 +1,93 @@
+    color: #414141;
+}
+
+nav{
```
```suggestion
nav {
```

zefrof commented on index.html:
```diff
@@ -1,7 +1,53 @@
 <!DOCTYPE html>
+<html class="no-js" lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/foundation.min.css">
+    <link rel="stylesheet" href="css/app.css">
+    <!-- Remove next link in final -->
+    <link rel="stylesheet" href="css/testing.css">
+    <!-- Below Calls to Google Fonts API -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&family=Questrial&display=swap" rel="stylesheet">
```
Should we move some/all of these to the footer?

justinw248 replied:
Those that we keep should be moved there, yes. I'll adjust before my next commit.

justinw248 committed:
Update css/app.css

This will need to be used later. We can delete for now, but more detailed changes will see it added back in.

Co-authored-by: Darren W <zefrof@protonmail.com>

justinw248 committed:
Update css/app.css

Co-authored-by: Darren W <zefrof@protonmail.com>
